app/exporters/base.py

```diff
@@ -8,6 +10,9 @@ def __init__(self, db: Session, job_id: str):
     def process(self) -> None:
         raise NotImplementedError()
 
+    def clean_download_folder(self, path: str) -> None:
+        shutil.rmtree(path, ignore_errors=True)
```

Comment:
I understand it should not rise an exception it deletion fails, but we should at least leave an information in the logs that it failed. Otherwise it could be not deleting files forever.


```python
>>> def onerror(function, path, excinfo):
...     print(f"While processing {path}")
...     traceback.print_exception(*excinfo)
... 
>>> shutil.rmtree("qwe", onerror=onerror)
While processing qwe/qwe
Traceback (most recent call last):
  File "/usr/lib/python3.8/shutil.py", line 670, in _rmtree_safe_fd
    os.unlink(entry.name, dir_fd=topfd)
PermissionError: [Errno 13] Permission denied: 'qwe'
While processing qwe
Traceback (most recent call last):
  File "/usr/lib/python3.8/shutil.py", line 717, in rmtree
    os.rmdir(path)
OSError: [Errno 39] Directory not empty: 'qwe'
```